### PR TITLE
Missing mandatory asterisk

### PR DESCRIPTION
**Pull request by SamuelTilly**

### CODE REVIEW

File: apps/webcert/src/feature/certificate/Question/QuestionHeaderAccordion.tsx [hunk 1]
```diff
@@ -56,7 +56,10 @@ const QuestionHeaderAccordion: React.FC<Props> = ({ config, displayMandatory, qu
           <Text className="iu-mb-300" dangerouslySetInnerHTML={sanitizeText(config.description)}></Text>
         </AccordionControl>
       ) : (
-        heading
+        <div>
+          {displayMandatory && <MandatoryIcon />}
```
Comment (ChrilleSigerhed) [inline apps/webcert/src/feature/certificate/Question/QuestionHeaderAccordion.tsx:60]: Lets add a test to verify this logic!

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
Added comment regarding test.

File: apps/webcert/src/feature/certificate/Question/QuestionHeaderAccordion.tsx (hunk 1 above)
Comment (SamuelTilly) [reply apps/webcert/src/feature/certificate/Question/QuestionHeaderAccordion.tsx:60]: Added a couple of tests

**Review by ChrilleSigerhed — APPROVED** [review]
Nice!

**Pull request merged**